integration-tests/kafka-oauth-keycloak/src/main/resources/application.properties

```diff
@@ -4,10 +4,7 @@ quarkus.log.category.\"org.apache.zookeeper\".level=WARN
 
 mp.messaging.connector.smallrye-kafka.security.protocol=SASL_PLAINTEXT
 mp.messaging.connector.smallrye-kafka.sasl.mechanism=OAUTHBEARER
-mp.messaging.connector.smallrye-kafka.sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required \
-  oauth.client.id="kafka-client" \
-  oauth.client.secret="kafka-client-secret" \
-  oauth.token.endpoint.uri="http://keycloak:8080/auth/realms/kafka-authz/protocol/openid-connect/token" ;
+mp.messaging.connector.smallrye-kafka.sasl.jaas.config=set_by_test
```

Comment:
Can you confirm that the configuration stay the same and you just moved it?

Reply:
It is now constructed in the test so that random port can be used.